tests/app/test_saver.py

```diff
@@ -75,13 +76,53 @@ def test_save_msg_audit_does_not_raise_exception_on_successful_save(self):
     def test_save_msg_audit_raises_message_save_exception_on_db_error(self):
         """Tests MessageSaveException generated if db commit fails saving message audit"""
         mock_session = mock.Mock(db.session)
-        mock_session.commit.side_effect = Exception("Not Saved")
+        mock_session.commit.side_effect = SQLAlchemyError("Not Saved")
         message_audit = {'msg_id': 'MsgId', 'msg_urn': 'Tej'}
         with self.app.app_context():
             with current_app.test_request_context():
                 with self.assertRaises(MessageSaveException):
                     Saver().save_msg_audit(message_audit['msg_id'], message_audit['msg_urn'], mock_session)
 
+    def test_save_actors_does_not_raise_exception_on_successful_save(self):
+        """Tests message actors is saved to database"""
+        message = {'msg_id': 'MsgId', 'msg_to': 'Tej', 'msg_from': 'some survey'}
+        with self.app.app_context():
+            with current_app.test_request_context():
+                Saver().save_message(SecureMessage(msg_id=message['msg_id']))
+                Saver().save_msg_actors(message['msg_id'], message['msg_to'], message['msg_from'], False)
+                self.assertTrue(True)
+
+    def test_save_actors_raises_message_save_exception_on_db_error(self):
+        """Tests MessageSaveException generated if db commit fails saving message actors"""
+        mock_session = mock.Mock(db.session)
+        mock_session.commit.side_effect = SQLAlchemyError("Not Saved")
+        message = {'msg_id': 'MsgId', 'msg_to': 'Tej', 'msg_from': 'someone'}
+        with self.app.app_context():
+            with current_app.test_request_context():
+                with self.assertRaises(MessageSaveException):
+                    Saver().save_msg_actors(message['msg_id'], message['msg_to'], message['msg_from'], False, mock_session)
+
+    def test_save_actors_saves_expected_information(self):
+        """Simple read write validation of actor table"""
+        pass
+        messages = [{'msg_id': 'MsgId1', 'msg_to': 'Tej', 'msg_from': 'some survey', 'sent_from_internal': True},
+                    {'msg_id': 'MsgId2', 'msg_to': '', 'msg_from': 'someone', 'sent_from_internal': False},
+                    {'msg_id': 'MsgId3', 'msg_to': 'aperson', 'msg_from': '', 'sent_from_internal': False}]
+        with self.app.app_context():
+            with current_app.test_request_context():
+                for message in messages:
+                    Saver().save_message(SecureMessage(message['msg_id']))
+                    Saver().save_msg_actors(message['msg_id'], message['msg_from'], message['msg_to'], message['sent_from_internal'])
+        with self.engine.connect() as con:
+            for message in messages:
+                query = "SELECT * FROM securemessage.actors WHERE msg_id='{}'".format(message['msg_id'])
```

Comment:
f string would be more succinct

Reply:
Fair cop , ( although we should also agree on this in the team ?) we should be consistent will change

Reply:
Changed as requested so will dismiss the review